src/api/ApiResponder.php

```diff
@@ -783,6 +783,121 @@ protected function get_interface_response_dismissGame($interface, $args) {
         return $retval;
     }
 
+    ////////////////////////////////////////////////////////////
+    // Tournament-related methods
+
+    /**
+     * Interface redirect for loadTourns
+     *
+     * @param BMInterface $interface
+     * @param array $args
+     * @return NULL|array
+     */
+    protected function get_interface_response_loadTourns($interface) {
+        return $interface->get_all_tourns($this->session_user_id());
+    }
+
+    /**
+     * Interface redirect for createTournament
+     *
+     * @param BMInterface $interface
+     * @param array $args
+     * @return NULL|array
+     */
+    protected function get_interface_response_createTournament($interface, $args) {
+        return $interface->tournament()->create_tournament(
+            $this->session_user_id(),
+            $args['tournamentType'],
+            $args['nPlayer'],
+            $args['maxWins'],
+            $args['description']
```

Comment:
A little bug: `description` is an optional property in the ApiSpec, so ApiResponder has to be able to handle the case where it's not sent.  As is, if the API client doesn't send it, we get an error:

```
PHP Notice:  Undefined index: description in /var/www/api/ApiResponder.php on line 813
```

and create_tournament blows up.

Reply:
Fixed